esi_leap/tests/api/controllers/v1/test_contract.py

```diff
@@ -62,6 +63,14 @@
 random_project = 'randomid'
 
 
+def create_test_contract_data():
+    return {
+   "uuid": "test_uuid",
+   "offer_uuid": "some-uuid",
```

Comment:
we should be consistent in how we write our fake uuids "-" or "_"

Reply:
oops that was a typo